src/main/java/salesforce/ui/pages/task/AbstractTask.java

```diff
@@ -84,6 +84,34 @@
      */
     protected abstract String getSubject(String subject);
 
+    /**
+     * Sets Contact.
+     *
+     * @param contact value.
+     */
+    protected abstract void setContact(String contact);
+
+    /**
+     * Returns contact.
+     *
+     * @return string value.
+     */
+    protected abstract String getContact();
+
+    /**
+     * Sets due date.
+     * @param dueDate value.
+     */
+    protected abstract void setDueDate(String dueDate);
```

Comment:
the same

Reply:
x2

Reply:
this is class abstract!!! checkstyle doesn't display error!